lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -291,7 +291,26 @@ void PatchEntryPointMutate::setupComputeWithCalls(Module *module) {
     if (func.isDeclaration() && func.getIntrinsicID() == Intrinsic::not_intrinsic &&
         !func.getName().startswith(lgcName::InternalCallPrefix) && !func.user_empty()) {
       m_computeWithCalls = true;
-      break;
+      return;
+    }
+
+    // Search for indirect calls
+    if (!func.isDeclaration()) {
```

Comment:
You don't need this "if"; for a declaration, the following loop has zero iterations.